WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Creation/CustomAmounts/AddCustomAmountViewModelTests.swift

```diff
@@ -2,6 +2,7 @@
 @testable import Yosemite
 import XCTest
 import Fakes
+import WooFoundation
```

Comment:
We don't seem to need this import:
```suggestion
```

Reply:
Nice catch! We don't want any not necessary imports. Done in 129a995afc6dfc5aa2c851873edbb8eceebd9340